BP/common/religion/religions/00_magyarism.txt

```diff
@@ -0,0 +1,212 @@
+﻿magyar_religion = {
```

Comment:
What is the justification for this separate religion?

Reply:
It was separate in CK3 and the culture/traditions of the Magyar appear to be noticeably different than those of other Finno-Urgic peoples (probably on account of exchange with tengrists in the plains).

Reply:
Pretty sure the Magyar are Tengri in WtWSMS, and I don't think it makes sense to distinguish them in 476, so they should probably stay whatever they were to minimise the amount of additional work for a group we have very little info about.

Reply:
Removed magyar_pagan with c47bd60.